onward/app/controllers/TechFeedbackController.scala

```diff
@@ -1,106 +0,0 @@
-package controllers
-
-import java.lang.Throwable
-
-import common._
-import conf.Configuration
-import model.Cached.RevalidatableResult
-import model.{ApplicationContext, Cached, MetaData, NoCache, SectionId, SimplePage}
-import play.api.data.Form
-import play.api.data.Forms._
-import play.api.libs.json.{JsValue, Json}
-import play.api.libs.ws._
-import play.api.mvc._
-
-import scala.concurrent.Future
-import scala.concurrent.duration._
-
-class TechFeedbackController(ws: WSClient, val controllerComponents: ControllerComponents)(implicit
-    context: ApplicationContext,
-) extends BaseController
-    with Logging
-    with ImplicitControllerExecutionContext {
-
-  def submitFeedback(path: String): Action[AnyContent] =
-    Action { implicit request =>
-      val feedbackForm = Form(
-        tuple(
-          "category" -> text,
-          "subject" -> text,
-          "body" -> text,
-          "user" -> text,
-          "extra" -> text,
-          "name" -> text,
-        ),
-      )
-
-      val (category, subject, body, user, extra, name) = feedbackForm.bindFromRequest.get
-
-      log.info(s"feedback submitted for category: $category")
-
-      val payload = Json.obj(
-        "category" -> java.net.URLEncoder.encode(category, "UTF-8"),
-        "subject" -> java.net.URLEncoder.encode(subject, "UTF-8"),
-        "body" -> java.net.URLEncoder.encode(body, "UTF-8"),
-        "user" -> java.net.URLEncoder.encode(user, "UTF-8"),
-        "extra" -> java.net.URLEncoder.encode(extra, "UTF-8"),
-        "name" -> java.net.URLEncoder.encode(name, "UTF-8"),
-      )
-
-      ws.url(Configuration.feedback.feedpipeEndpoint)
```

Comment:
It would be great to remove all the config items used in this file if they're now unused

Reply:
👍 Removed. Should I also find where these values are stored in the AWS account and delete them?

Reply:
delete them... when this gets merged I mean.

Reply:
If we delete them then we lose all the history of those parameters, which we may not want to do, so I'm not sure. Maybe send an email to dotcom.platform and we can take it on if something needs to be done?